templates/VTEX - Template openAPI.json

```diff
@@ -160,19 +160,30 @@
         {
             "appKey": [],
             "appToken": []
+        },
+        {
+            "VtexIdclientAutCookie": []
         }
     ],
     "components": {
         "securitySchemes": {
             "appKey": {
                 "type": "apiKey",
                 "in": "header",
-                "name": "X-VTEX-API-AppKey"
+                "name": "X-VTEX-API-AppKey",
+                "description": "Unique identifier of the [application key](https://developers.vtex.com/docs/guides/authentication-overview#application-keys)."
             },
             "appToken": {
                 "type": "apiKey",
                 "in": "header",
-                "name": "X-VTEX-API-AppToken"
+                "name": "X-VTEX-API-AppToken",
+                "description": "Secret token of the [application key](https://developers.vtex.com/docs/guides/authentication-overview#application-keys)."
+            },
+            "VtexIdclientAutCookie": {
+                "type": "apiKey",
```

Comment:
Tecnicamente não é uma "apiKey". Essa nomenclatura é do Open API? Se for funcional, tranquilo, mas se for semântico, acho que vale revisar.

Reply:
Eu fiquei na dúvida mas achei que tá mais próximo do [conceito de apiKey no openapi](https://swagger.io/docs/specification/authentication/api-keys/), já que pode passar como um header. A alternativa seria [cookie](https://swagger.io/docs/specification/authentication/cookie-authentication/), mas nesse caso o nome do header teria que ser Set-Cookie, que não é como se usa o VtexIdclientAutCookie... 👀

Reply:
Hm, agora que li, acho que na verdade é cookie mesmo. Note que dentre essas [etapas que o Swagger enumera](https://swagger.io/docs/specification/authentication/cookie-authentication/), a integração do lojista está cumprindo a etapa `3` (trocando a chave de `Cookie` para `VtexIdclientAutCookie`).

O `Set-cookie` é um header que a VTEX manda para o cliente após o login do usuário, cujo valor é o user token.

Reply:
Aliás, acho que seria tipo
`Cookie: VtexIdclientAutCookie={user-token}`

Reply:
Seguindo o [padrão do Swagger para cookie](https://swagger.io/docs/specification/authentication/cookie-authentication/), continuaria sendo uma apiKey, mas em vez de header, cookie (L184):

```json
                "in": "cookie",
```